### ISSUE TITLE
Slots of Subjects are not updated

### ISSUE THREAD

**Issue by debajyotidasgupta** [metakgp/what-slot#12]
I was going through the slots for course CS30002, which is the operating systems course offered by the Computer Science and Engineering department. The slots for this course, as shown by the applications, are as follows.

![image](https://user-images.githubusercontent.com/23581364/103134306-489b1200-46d6-11eb-99ac-b3cdbec29b75.png)

However, as per the current schedule given in ERP, the CS30002 course has been allotted the following slot.

![image](https://user-images.githubusercontent.com/23581364/103134372-c4955a00-46d6-11eb-9758-37741f4415e9.png)

Which as per the spring semester schedule refers to the following slots.

![image](https://user-images.githubusercontent.com/23581364/103134403-08885f00-46d7-11eb-8724-dfb15513d3c0.png)

So basically the following are the differences in the slots that arises from the updated time table. 

![image](https://user-images.githubusercontent.com/23581364/103134455-4eddbe00-46d7-11eb-9bdf-6ba33673f3c7.png)

This also arises a second issue, that is the professors who are teaching the course. As per the application, professors teaching the course are.

![image](https://user-images.githubusercontent.com/23581364/103134500-86e50100-46d7-11eb-83cd-0321c1cf5b64.png)

However, as per the updated time table in ERP, the professors taking the course are.

![image](https://user-images.githubusercontent.com/23581364/103134533-c3186180-46d7-11eb-9330-f9b72bd0d75d.png)

@thealphadollar @xypnox please look into it 

Thank You

**Comment by thealphadollar** [comment]
Hey, @debajyotidasgupta thanks for bringing it up and creating an issue. The problem, as far as I understand, is that we have not updated the course list we are using. Would you like to try (the instructions are provided in the [README.md](https://github.com/metakgp/what-slot#updating-for-new-semester)) and raise a PR with the updated subject list?

**Comment by debajyotidasgupta** [comment]
@thealphadollar please look into the PR #13 which I believe should address this issue.
